nimble-newts/askgrieves/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM python:3.8.5-buster
```

Comment:
Good use of Docker!

If you copy the Pipfile & lockfile separately you can make use of Docker cache to speed up build times. The ideal Dockerfile would go like:

- Install `pipenv`
- Copy `Pipfile` and `Pipfile.lock` to the directory
- Install dependencies
- Run all other commands

This means that pipenv does not have to be reinstalled if you change dependencies because it was built in an earlier layer where dependencies were irrelevant.